packages/block-library/src/tag-cloud/editor.scss

```diff
@@ -7,3 +7,11 @@
 	margin: 0;
 	padding: 0;
 }
+
+.wp-block-tag-cloud__inspector-settings {
+	.components-base-control,
+	.components-base-control:last-child {
+		// Cancel out extra margins added by block inspector
+		margin-bottom: 0;
+	}
+}
```

Comment:
So painful 😭 I'm going to propose a better way to deal with these once the deprecation work is done.

Reply:
Indeed. Let's definitely open an issue to keep track of those 👍